Sources/NIOHTTP2/HTTP2StreamMultiplexer.swift

```diff
@@ -205,6 +205,15 @@ public final class HTTP2StreamMultiplexer: ChannelInboundHandler, ChannelOutboun
         context.fireChannelWritabilityChanged()
     }
 
+    public func errorCaught(context: ChannelHandlerContext, error: Error) {
+        if let streamError = error as? NIOHTTP2Errors.StreamError,
+           let channel = self.streams[streamError.streamID] {
+            channel.receiveStreamError(streamError)
+        }
```

Comment:
Is it possible to receive a stream error for a stream we don't know about? Wondering whether we should drop stream errors for streams we aren't aware of (to avoid downstream handlers, for example, closing the channel on receiving an error).

Reply:
Yes, it is, but I don't think we should drop them. Just because the multiplexer doesn't know about them doesn't mean the error is not of interest.